src/components/legacy/grid/grid.tsx

```diff
@@ -25,7 +25,7 @@ export const Grid = styled.div<GridProps>`
   padding: 0
     ${(props) => (props.disableGap || props.nested ? 0 : GRID_GAP_MOBILE)}; // outer gap
 
-  max-width: ${(props) => props.theme.maxWidth};
+  // max-width: ${(props) => props.theme.maxWidth};
```

Comment:
Is that commented out on purpose?

Reply:
it should be deleted, but it is also for the navigation bar, since we decided to make it in extra PR, I will revert this changes